dataprofiler/profilers/float_column_profile.py

```diff
@@ -50,6 +50,14 @@ def __add__(self, other):
         merged_profile = FloatColumn(None)
         BaseColumnPrimitiveTypeProfiler._add_helper(merged_profile, self, other)
         NumericStatsMixin._add_helper(merged_profile, self, other)
+        
+        self._merge_calculations(merged_profile.__calculations,
+                                 self.__calculations,
+                                 other.__calculations)
+
+        if "precision" in merged_profile.__calculations:
+            merged_profile.precision = min(self.precision, other.precision)
```

Comment:
Is there a reason this isn't done in _merge_calculations?

Reply:
Nvm, explained over standup mtg (_merge_calculations is generic and top level and just for instantiating dict, not populating it)